pulp_rpm/app/viewsets.py

```diff
@@ -97,7 +99,7 @@ class Meta:
         }
 
 
-class PackageViewSet(SingleArtifactContentUploadViewSet):
+class PackageViewSet(SingleArtifactContentUploadViewSet, RolesMixin):
```

Comment:
Probably not needed unless we will have permission checking on content directly.